bot/exts/fun/quack.py

```diff
@@ -0,0 +1,75 @@
+import random
+from typing import Optional
+
+import discord
+from aiohttp import client_exceptions
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quackstack(commands.Cog):
+    """Cog used for wrapping Quackstack."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, ducktype: str = "duck", *, seed: Optional[str] = None) -> None:
+        """
+        Uses the Quackstack API to generate a random duck.
+
+        If a seed is provided, a duck is generated based on the given seed.
+        If a duck type is provided, either a duck or manduck will be generated.
+        """
+        error_embed = discord.Embed(
+            title=random.choice(NEGATIVE_REPLIES),
+            description="The request failed. Please try again later.",
+            color=Colours.soft_red,
+        )
+
+        ducktype = ducktype.lower()
+        if ducktype not in ["duck", "manduck"]:
+            raise commands.BadArgument(
+                "The duck type can only be `duck` or `manduck`."
+            )
+
+        quackstack_url = f"{API_URL}/{ducktype}"
+        params = {}
+        if seed is not None:
+            try:
+                seed = int(seed)
+            except ValueError:
+                # Converts any words or words with numbers into an integer using ord()
```

Comment:
Is this comment talking about how `str.encode()` converts strings into integers? Either way, this isn't using the `ord()` function, but the string's `encode()` method.

```suggestion
                # Converts any word(s) with numbers into an integer
```